Description: fix ffi.rake path for building
 Building in separate directories, fix path for those.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: not-needed
Last-Update: 2023-12-26

---

--- protobuf-3.25.1.orig/ruby/ext/google/protobuf_c/Rakefile
+++ protobuf-3.25.1/ruby/ext/google/protobuf_c/Rakefile
@@ -1,3 +1,3 @@
-import '../../../lib/google/tasks/ffi.rake'
+import '../../../../../../../../../../../../../ruby/lib/google/tasks/ffi.rake'
 
-task default: ['ffi-protobuf:default']
\ No newline at end of file
+task default: ['ffi-protobuf:default']
